routing-lab/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './App.css';
+import {Route} from 'react-router-dom'
+import NavBar from './NavBar';
+import RandDogImg from './Component/RandDogImg'
+import MultDogs from './Component/MultDogs';
+import RandCat from './Component/RandCat'
+import MultCats from './Component/MultCats'
+import RandDogBreed from './Component/RandDogBreed'
+function App() {
+  return (
+    <div className="App">
+      Cat/Dog API
+      <NavBar/>
+     
+
```

Comment:
Wrapping all of these routes in a `<Switch>` component allows you to add a route specifically for error handling